images/tool/Dockerfile

```diff
@@ -16,7 +16,10 @@ FROM golang:1.20.2
 
 # Required input: -e TOOL_NAME=name, i.e. TOOL_NAME=flaky-test-reporter
 ARG TOOL_NAME
+ARG COMMIT_HASH
```

Comment:
Also, what is the use of this? I don't see you using it anywhere other than using it as an ENV var?

Reply:
It is passed by Makefile. The build warns if the `ARG` isn't used. It's just used in env variable, so we can inspect it - maybe better to use `LABEL` stanza, instead?

Reply:
I would add some explanation in comment and maybe remove the ENV that is not used

Reply:
The hack scripts use `COMMIT_HASH` to print the commit details of the prow-tests image. helpful for debugging broken tooling.

https://github.com/knative/test-infra/pull/3025 for details